[2023-05-19 15:25:46,026] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [queued]>
[2023-05-19 15:25:46,034] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [queued]>
[2023-05-19 15:25:46,034] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 15:25:46,035] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-19 15:25:46,035] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 15:25:46,048] {taskinstance.py:1377} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2023-05-16 00:00:00+00:00
[2023-05-19 15:25:46,051] {standard_task_runner.py:52} INFO - Started process 3312 to run task
[2023-05-19 15:25:46,059] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'TwitterDAG', 'twitter_datascience', 'scheduled__2023-05-16T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpbh99aehb', '--error-file', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpg581a6kp']
[2023-05-19 15:25:46,061] {standard_task_runner.py:80} INFO - Job 6: Subtask twitter_datascience
[2023-05-19 15:25:46,134] {task_command.py:370} INFO - Running <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [running]> on host MacBook-Air-de-Mariana-2.local
[2023-05-19 15:25:46,428] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=TwitterDAG
AIRFLOW_CTX_TASK_ID=twitter_datascience
AIRFLOW_CTX_EXECUTION_DATE=2023-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-16T00:00:00+00:00
[2023-05-19 15:25:46,442] {base.py:68} INFO - Using connection ID 'twitter_default' for task execution.
[2023-05-19 15:25:46,444] {twitter_hook.py:34} INFO - URL: https://labdados.com/2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2023-05-16T00:00:00.00Z&end_time=2023-05-17T00:00:00.00Z
[2023-05-19 15:25:47,040] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=TwitterDAG, task_id=twitter_datascience, execution_date=20230516T000000, start_date=20230519T182546, end_date=20230519T182547
[2023-05-19 15:25:47,088] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-19 15:25:47,131] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-19 18:19:40,997] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [queued]>
[2023-05-19 18:19:41,012] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [queued]>
[2023-05-19 18:19:41,012] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 18:19:41,012] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-19 18:19:41,012] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-19 18:19:41,043] {taskinstance.py:1377} INFO - Executing <Task(TwitterOperator): twitter_datascience> on 2023-05-16 00:00:00+00:00
[2023-05-19 18:19:41,046] {standard_task_runner.py:52} INFO - Started process 9472 to run task
[2023-05-19 18:19:41,054] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'TwitterDAG', 'twitter_datascience', 'scheduled__2023-05-16T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/twitter_dag.py', '--cfg-path', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpo39bv42q', '--error-file', '/var/folders/4m/2vllbq951394clq6jtlsj0p40000gn/T/tmpy69m6e6y']
[2023-05-19 18:19:41,057] {standard_task_runner.py:80} INFO - Job 37: Subtask twitter_datascience
[2023-05-19 18:19:41,152] {task_command.py:370} INFO - Running <TaskInstance: TwitterDAG.twitter_datascience scheduled__2023-05-16T00:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2023-05-19 18:19:41,259] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=TwitterDAG
AIRFLOW_CTX_TASK_ID=twitter_datascience
AIRFLOW_CTX_EXECUTION_DATE=2023-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-16T00:00:00+00:00
[2023-05-19 18:19:41,271] {base.py:68} INFO - Using connection ID 'twitter_default' for task execution.
[2023-05-19 18:19:41,276] {twitter_hook.py:34} INFO - URL: https://labdados.com/2/tweets/search/recent?query=datascience&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,lang,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2023-05-16T00:00:00.00Z&end_time=2023-05-17T00:00:00.00Z
[2023-05-19 18:19:43,052] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=TwitterDAG, task_id=twitter_datascience, execution_date=20230516T000000, start_date=20230519T211940, end_date=20230519T211943
[2023-05-19 18:19:43,096] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-19 18:19:43,151] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
